mthca: Events received for non-existent QPs should generate a warning.

Events received for non-existent QPs should generate a warning along with
the event type which was received.

Signed-off-by: Jack Morgenstein <jackm@dev.mellanox.co.il>

---
 drivers/infiniband/hw/mthca/mthca_qp.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: ofed_kernel/drivers/infiniband/hw/mthca/mthca_qp.c
===================================================================
--- ofed_kernel.orig/drivers/infiniband/hw/mthca/mthca_qp.c
+++ ofed_kernel/drivers/infiniband/hw/mthca/mthca_qp.c
@@ -247,7 +247,8 @@ void mthca_qp_event(struct mthca_dev *de
 	spin_unlock(&dev->qp_table.lock);
 
 	if (!qp) {
-		mthca_warn(dev, "Async event for bogus QP %08x\n", qpn);
+		mthca_warn(dev, "Async event %d for bogus QP %08x\n",
+			  (int) event_type, qpn);
 		return;
 	}
 
